config/test/patch_manager_deployment.yaml

```diff
@@ -17,4 +17,4 @@ spec:
         envFrom: []
         name: manager
         image: ${METALSTACK_IMAGE}
-      hostNetwork: true
\ No newline at end of file
+      hostNetwork: false
```

Comment:
`hostNetwork` defaults to `false`. It's up to you if you want to make your intent obvious.